mod/tagger.php

```diff
@@ -121,7 +121,7 @@ function tagger_content(App $a) {
 	$arr['contact-id'] = $contact['id'];
 	$arr['type'] = 'activity';
 	$arr['wall'] = $item['wall'];
-	$arr['gravity'] = GRAVITY_COMMENT;
+	$arr['gravity'] = GRAVITY_ACTIVITY;
```

Comment:
Does this effectively hide this kind of posts?

Reply:
I think so.

Reply:
I think it should still be shown because there's no other universal way of showing that a tag was added to a post. The label display can be disabled (like I do, although it creeps back in on certain pages).

Reply:
I think it should still be shown because there's no other universal way of showing that a tag was added to a post. The label display can be disabled (like I do, although it creeps back in on certain pages).

Reply:
For me this always looked like useless information. But since I want to move on to the next steps (the tags), I will change this back.